src/main/java/com/stuypulse/robot/commands/auton/FiveBallAuton.java

```diff
@@ -0,0 +1,108 @@
+/************************ PROJECT DORCAS ************************/
+/* Copyright (c) 2022 StuyPulse Robotics. All rights reserved.  */
+/* This work is licensed under the terms of the MIT license.    */
+/****************************************************************/
+
+package com.stuypulse.robot.commands.auton;
+
+import com.stuypulse.robot.RobotContainer;
+import com.stuypulse.robot.commands.conveyor.ConveyorShootCommand;
+import com.stuypulse.robot.commands.drivetrain.DrivetrainAlignCommand;
+import com.stuypulse.robot.commands.drivetrain.DrivetrainRamseteCommand;
+import com.stuypulse.robot.commands.intake.IntakeAcquireCommand;
+import com.stuypulse.robot.commands.intake.IntakeExtendCommand;
+import com.stuypulse.robot.commands.leds.LEDSetCommand;
+import com.stuypulse.robot.commands.shooter.ShooterRingShotCommand;
+import com.stuypulse.robot.subsystems.LEDController.LEDColor;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+/*-
+ * @author Vincent Wang (vinowang921@gmail.com)
+ * @author Ivan Wei (ivanw8288@gmail.com)
+ * @author Ivan Chen (ivanchen07@gmail.com)
+ * @author Eric Lin (ericlin071906@gmail.com)
+ * @author Marc Jiang (mjiang05@gmail.com)
+ * @author Ian Jiang (ijiang05@gmail.com)
+ * @author Carmin Vuong (carminvuong@gmail.com)
+ * @author Samuel Chen(samchen1738@gmail.com)
+ */
+
+public class FiveBallAuton extends SequentialCommandGroup {
+
+    // private static final int
+    // Time it takes for the shooter to reach the target speed
+    private static final int SHOOTER_INITIALIZE_DELAY = 1;
+    // Time it takes for the robot.conveyor to give the shooter the ball
+    private static final int CONVEYOR_TO_SHOOTER = 1;
+    // Time we want to give the robot.drivetrain to align
+    private static final int DRIVETRAIN_ALIGN_TIME = 2;
+
+    private static final int HUMAN_WAIT_TIME = 2;
+
+    // Timeouts for Pathweaver paths
+    private static final int FIVE_BALL_TO_SECOND_BALL_TIMEOUT = 10;
+    private static final int FIVE_BALL_TO_TERMINAL_TIMEOUT = 10;
+    private static final int FIVE_BALL_TERMINAL_TO_SHOOT_TIMEOUT = 10;
+    private static final int FIVE_BALL_TO_WALL_BALL_TIMEOUT = 10;
```

Comment:
the ramsete commands are on a timer anyway, no need to add a with timeout. the alignment and shooting do need them